fastlane/Fastfile

```diff
@@ -41,6 +41,7 @@ end
 
 lane :ci do
   lint
+  test
```

Comment:
I would move the running of tests step after the building step :)

Reply:
Ok, I do not have a strong opinion about that. Changed ✅ 

In fact as soon as we have tests for both targets, the `test` step will make redundant the building steps, because it will build the App to run the apps.

But that is a next step...